src/main/java/io/kestra/plugin/weaviate/Query.java

```diff
@@ -61,14 +64,14 @@ public class Query extends WeaviateConnection implements RunnableTask<Query.Outp
     @NotBlank
     @PluginProperty(dynamic = true)
     @NotBlank
-    protected String query;
+    private String query;
 
     @Schema(
         title = "Whether store data in internal storage. Default is falses"
     )
     @PluginProperty
     @Builder.Default
-    protected boolean store = false;
+    private boolean store = false;
 
     @Override
     public Query.Output run(RunContext runContext) throws Exception {
```

Comment:
Please use FetchOutput as your output like [here](https://github.com/kestra-io/plugin-aws/blob/master/src/main/java/io/kestra/plugin/aws/dynamodb/Query.java#L90C14-L90C14)

Reply:
Also please add a fetchType parameter like [here](https://github.com/kestra-io/plugin-aws/blob/master/src/main/java/io/kestra/plugin/aws/dynamodb/Query.java#L50-L59) and change the output based on this

Reply:
FETCH_ONE will populate row from FetchOutput only (you retrieve only the first row)
FETCH will retrieve every objects and add them to rows in FetchOutput
STORE will create an URI from internal storage with objects stored in it (as you did already), populate the uri field
NONE will do nothing